govcd/nsxv_nat_test.go

```diff
@@ -131,6 +131,17 @@ func testNsxvNat(natRule *types.EdgeNatRule, vcd *TestVCD, check *C, edge EdgeGa
 
 	check.Assert(updatedNatRule.Description, Equals, natRule.Description)
 
+	// Test that we can extract a list of NSXV NAT rules, and that one of them is the rule we have got when searching by ID
+	natRules, err := edge.GetNsxvNatRules()
+	check.Assert(err, IsNil)
```

Comment:
check that `natRules` is not nill is missing

Reply:
done